src/components/cursor-controller.js

```diff
@@ -97,6 +98,11 @@ AFRAME.registerComponent("cursor-controller", {
       if (this.data.drawLine) {
         this.el.setAttribute("line", { start: this.origin.clone(), end: this.data.cursor.object3D.position.clone() });
       }
+
+      // The curser will always be oriented towards the player about its Y axis, so bjects held by the cursor will rotate towards the player.
```

Comment:
typos:
// The ~~curser~~ cursor will always be oriented towards the player about its Y axis, so ~~bjects~~ objects held by the cursor will rotate towards the player.